contracts/claim/assembly/Claim.ts

```diff
@@ -17,7 +17,7 @@ namespace State {
 }
 
 namespace Constants {
-  export const KOIN_CONTRACT_ID = BUILD_FOR_TESTING ? Base58.decode('1BRmrUgtSQVUggoeE9weG4f7nidyydnYfQ') : Base58.decode('19JntSm8pSNETT9aHTwAUHC5RMoaSmgZPJ');
+  export const KOIN_CONTRACT_ID = System.getContractAddress('koin');
```

Comment:
This should be done using the static and getter method pattern so that we only get the contract address if we actually need it.

Or, if we only need it in one place (probably the case), then let's just get it there. It doesn't need to be a constant.